core/linux-dist/evdev.cpp

```diff
@@ -353,7 +355,7 @@
 					} else if (ie.code == controller->mapping->Btn_Start) {
 						SET_FLAG(kcode[port], DC_BTN_START, ie.value);
 					} else if (ie.code == controller->mapping->Btn_Escape) {
-						die("death by escape key");
+                                                start_shutdown();
```

Comment:
Indentation.

I would also like to add the string to ``start_shutdown()`` to differntiate between ``"death by escape key"`` (evdev) and ``"X11: death by escape key"`` (X11). But that's a minor nit and I'm fine either way.